packages/koa/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@hatchifyjs/koa",
+  "private": true,
+  "version": "0.0.8",
+  "description": "Fullstack CRUD Application Solution",
+  "main": "dist/exports.js",
+  "types": "dist/exports.d.ts",
+  "eslintConfig": {
+    "extends": [
+      "../../package.json"
+    ],
+    "ignorePatterns": [
+      "dist/*"
+    ],
+    "overrides": [
+      {
+        "files": [
+          "*.ts"
+        ],
+        "rules": {
+          "@typescript-eslint/explicit-module-boundary-types": "warn"
+        }
+      }
+    ]
+  },
+  "scripts": {
+    "prebuild": "npm run prettier:fix",
+    "prepare": "npm run build",
+    "build": "tsc",
+    "build:watch": "tsc --watch",
+    "clean": "rm -rf tsconfig.tsbuildinfo dist",
+    "depcheck": "depcheck .",
+    "test": "jest",
+    "test:coverage": "jest --watch --collect-coverage",
+    "docs": "typedoc",
+    "docs:watch": "typedoc --watch",
+    "test:watch": "jest --watch",
+    "test:verbose": "jest --watch --verbose",
+    "typecheck": "tsc --noEmit",
+    "eslint": "eslint .",
+    "eslint:fix": "eslint --fix .",
+    "prettier": "prettier --check .",
+    "prettier:fix": "prettier --write .",
+    "prepublishOnly": "npm run build"
+  },
```

Comment:
I would expect that we are leveraging nx's build system rather than npm scripts.

Reply:
Disregard